build.xml

```diff
@@ -728,6 +728,20 @@
     <!--                                                                                    -->
 
     <!-- Internal use -->
+    <target name="gradle-install" description="Build and install (in maven local) the uPortal modules that have been ported to a Gradle-based build">
+        <!--
+         | TEMPORARY MEASURE!
```

Comment:
This was a temporary measure for before move to gradle, can this be removed now?